pages/docs/quickstart.mdx

```diff
@@ -138,7 +149,11 @@ The first dropdown for the `base` repository (destination) should be set to `Edd
 ![pull request](https://user-images.githubusercontent.com/82668196/208292196-c374d328-6bed-4f7b-adbc-b55c29b7c367.png)
 
 18. You will receive a GitHub notification when you have a comment, review or your Pull Request has been merged
+
+<div className="fullwidth-breakwords">
```

Comment:
people wouldn't know when to add this, I think this method is too confusing

Reply:
Sure i will come more optimised and clean code. I will hunt down this issue 🦈✊

Reply:
Thank you @SyedImam1998 👍